.github/workflows/create.yml

```diff
@@ -28,6 +28,12 @@ jobs:
             cibw_skip: '*-manylinux_i686'
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Set up QEMU for aarch64 wheels
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: arm64
```

Comment:
QEMU only needs to be set up for `aarch64` builds, not also `x86_64` ones.
We're doing something similar here: https://github.com/kivy/kivy/blob/master/.github/workflows/manylinux_wheels.yml

(Same applies for the `Build and test aarch64 wheels`) step.